packages/experiments/src/components/PersonaCoin/PersonaCoin.styles.ts

```diff
@@ -15,7 +15,7 @@ export const PersonaCoinStyles: IPersonaCoinComponent['styles'] = props => {
   return {
     root: {
       position: 'relative',
-      backgroundColor: coinColor,
+      backgroundColor: props.isPictureLoaded ? 'transparent' : coinColor,
```

Comment:
```suggestion
      backgroundColor: props.isPictureLoaded ? undefined : coinColor,
```

Perhaps? I haven't tested, but if all we're doing is trying to eliminate the `background-color` it _should_ work.

Reply:
Yep that works :D